app.css

```diff
@@ -1,3 +1,210 @@
 * {
   box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+.banner {
+  max-width: auto;
+  height: 19rem;
+  background-image: url(http://lorempixel.com/400/200);
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 auto;
+  max-width: 55rem;
+  padding-top: 1rem;
+}
+
+.links {
+  display: flex;
+}
+.link__item {
+  margin: 0 auto;
+  width: 5rem;
+}
+
+.navbar__logo {
+
+}
+
+.hero {
+  text-align: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.about {
+
+}
+
+.search {
+
+}
+
+.title {
+  font-size: 60px;
+  color: white;
+}
+
+.subtitle {
+}
+
+/* Left Article */
+.left-article {
+  display: flex;
+  flex-direction: column;
+  margin-left: 5rem;
+  justify-content: space-between;
+  max-width: 70rem;
+  margin-top: 3rem;
+  height: 25rem;
```

Comment:
Don't set hard height on elements. This makes the design break as content changes.